javav2/example_code/kms/metadata.yaml

```diff
@@ -0,0 +1,44 @@
+--- 
+files:
+  - path: src/main/java/com/example/kms/CreateAlias.java
+    services:
+      - kms
+  - path: src/main/java/com/example/redshift/CreateCustomerKey.java
+    services:
+      - kms
+  - path: src/main/java/com/example/kms/CreateGrant.java
+    services:
+      - kms
+  - path: src/main/java/com/example/kms/DeleteAlias.java
+    services:
+      - kms
+  - path: src/test/java/example/kms/DescribeKey.java
```

Comment:
A bunch of the paths in this file are wrong. For example, this one should be: src\main\java\com\example\kms\DescribeKey.java.

Reply:
Fixed these

Reply:
Verified.